staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -928,6 +1046,9 @@ func (c *cacheWatcher) Stop() {
 	c.forget()
 }
 
+// TODO
+// stop() is protected by Cacher.Lock(), rename it to
+// stopThreadUnsafe and remove the sync.Mutex.
```

Comment:
Let's try to keep TODOs out of merged code. Is this something we can resolve in this PR? If not would it be appropriate to open an issue to track it?

Reply:
I would like to avoid mixing these changes with this PR - this todo can be attached to this issue:
https://github.com/kubernetes/kubernetes/issues/73958

TODO(#73958): stop() is ....

Reply:
thanks, add ref#73958